src/commands/deploy.ts

```diff
@@ -3,28 +3,87 @@
  *  Licensed under the MIT License. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 
-// tslint:disable-next-line:no-require-imports
-import WebSiteManagementClient = require('azure-arm-website');
+import * as fse from 'fs-extra';
+import * as path from 'path';
 import * as vscode from 'vscode';
+import { SiteWrapper } from 'vscode-azureappservice';
 import { AzureFunctionsExplorer } from '../AzureFunctionsExplorer';
-import { IUserInterface } from '../IUserInterface';
+import { NoPackagedJavaFunctionError } from '../errors';
+import { IUserInterface, Pick } from '../IUserInterface';
 import { localize } from '../localize';
 import { FunctionAppNode } from '../nodes/FunctionAppNode';
 import { getWebSiteClient } from '../nodes/SubscriptionNode';
+import { TemplateLanguage } from '../templates/Template';
+import { cpUtils } from '../utils/cpUtils';
+import { getProjectType } from '../utils/project';
 import * as workspaceUtil from '../utils/workspace';
 import { VSCodeUI } from '../VSCodeUI';
 
+/* tslint:disable:no-require-imports */
```

Comment:
Don't disable this for the entire file. We still prefer non-require imports where possible

And keep all imports in the same grouping (they're automatically alphabetized when you save the file, but only if they're together)